microsoft-365/managed-desktop/get-started/project-visio.md

```diff
@@ -23,26 +23,17 @@ Admins should verify that they meet these prerequisites:
 - **Company Portal** -  The Company Portal must be available in your tenant for your users to install these applications. If the Company Portal isn’t deployed in your tenant, see [Company Portal](company-portal.md).
 
 ## Deploy Project and Visio for Microsoft Managed Desktop devices
-After you submit your support request, Microsoft Managed Desktop will create three Azure AD groups and three application deployments through Microsoft Intune to deploy the apps to your tenant.  
+Microsoft Managed Desktop will add Microsoft Project and Microsoft Visio as two Win32 Applications in Microsoft Intune. We will also create two groups in Azure Active Directory which will be assigned to the coresponding application with the "Available" intent. 
```

Comment:
```suggestion
Microsoft Managed Desktop will add Microsoft Project and Microsoft Visio as two Win32 Applications in Microsoft Intune. We will also create two groups in Azure Active Directory which will be assigned to the corresponding application with the _Available_ intent. 
```

Reply:
Thanks for this good catches, the other edits we should make available bold and change "intent" to "assignment type" per the intune language: 

We will also create two groups in Azure Active Directory which will be assigned to the corresponding application with the **Available** assignment type.